packages/kos-sdk/src/chains/tron/requests.rs

```diff
@@ -66,13 +74,21 @@ pub async fn create_asset_transfer(
     create_transaction(url, contract).await
 }
 
+pub async fn create_trc20_transfer(
```

Comment:
lets use `trigger_smartcontract` as this is not only for trc20 trigger request build